src/js/components/Breadcrumb.js

```diff
@@ -46,6 +72,12 @@ class Breadcrumb extends React.Component {
   }
 
   shouldComponentUpdate(nextProps, nextState) {
+    let {buildCrumbCustomParams} = this.props;
+
+    if (!DeepEqual(nextProps.buildCrumbCustomParams, buildCrumbCustomParams)) {
```

Comment:
we need it because `nextState.availableWidth == null || nextState.expandedWidth == null` will return `false` before the last part of this method has a chance to return `true`